docs/40_smart-contract-guides/92_read-only-queries.md

```diff
@@ -0,0 +1,337 @@
+
+## Call the Read-Only Query 
+Once you have deployed the smart contract you can test it by calling the actions and seeing the data returned. Assuming the smart contract is deployed to an account called _scholder_ and that you have an account named _bob_ to authorise calling the actions. 
+
+* Call get():
+
+```shell
+cleos push transaction --read-only -j '{"actions":[{"account":"scholder","name":"get","authorization":[{"actor":"bob","permission":"active"}],"data":""}]}' -p bob@active
+```
+
+The output should be similar to:
+
+```json
+{
+  "head_block_num": 89230,
+  "head_block_id": "00015c8e67e66b8bb1daeeb6452fefea22509d0ef99381f34ed4b1486036bdbd",
+  "last_irreversible_block_num": 89229,
+  "last_irreversible_block_id": "00015c8d117aa070cea8bbe8bae16d27cfa321d54deaa89b2125b5d2f5a4922b",
+  "code_hash": "89959d556cf0a0863064269f93dbc69c3b1e3b278385d495fc59f3509f77a6fb",
+  "pending_transactions": [],
+  "result": {
+    "id": "5394a83aebf2c43dd007473bd917ef4d7461c73395b1b504df6b853787f4967f",
+    "block_num": 89231,
+    "block_time": "2021-06-03T09:46:39.500",
+    "producer_block_id": null,
+    "receipt": {
+      "status": "executed",
+      "cpu_usage_us": 126,
+      "net_usage_words": 12
+    },
+    "elapsed": 3562,
+    "net_usage": 96,
+    "scheduled": false,
+    "action_traces": [{
+        "action_ordinal": 1,
+        "creator_action_ordinal": 0,
+        "closest_unnotified_ancestor_action_ordinal": 0,
+        "receipt": {
+          "receiver": "scholder",
+          "act_digest": "ffe396998865e412f71cb314f5b4fec9a05cb7a043f2bcc38e3e5bff70f6cfe7",
+          "global_sequence": 89247,
+          "recv_sequence": 6,
+          "auth_sequence": [[
+              "bob",
+              6
+            ]
+          ],
+          "code_sequence": 2,
+          "abi_sequence": 2
+        },
+        "receiver": "scholder",
+        "act": {
+          "account": "scholder",
+          "name": "get",
+          "authorization": [{
+              "actor": "bob",
+              "permission": "active"
+            }
+          ],
+          "hex_data": ""
+        },
+        "context_free": false,
+        "elapsed": 3480,
+        "console": "",
+        "trx_id": "5394a83aebf2c43dd007473bd917ef4d7461c73395b1b504df6b853787f4967f",
+        "block_num": 89231,
+        "block_time": "2021-06-03T09:46:39.500",
+        "producer_block_id": null,
+        "account_ram_deltas": [],
+        "account_disk_deltas": [],
+        "except": null,
+        "error_code": null,
+        "return_value_hex_data": "045068696c0100000033000000",
+        "return_value_data": "Phil"
+      }
+    ],
+    "account_ram_delta": null,
+    "except": null,
+    "error_code": null
+  }
+}
+```
+
+* Call getComp():
+
+```shell
+cleos push transaction --read-only -j '{"actions":[{"account":"scholder","name":"getComp","authorization":[{"actor":"bob","permission":"active"}],"data":""}]}' -p bob@active
+```
+
+The output should be similar to:
```

Comment:
'should' vs 'will'

Reply:
imperative is deemed incorrect/rude and I have been pulled up for using will before.